gui/wxpython/mapdisp/main.py

```diff
@@ -13,7 +13,8 @@
  - mapdisp::MapApp
 
 Usage:
-python mapdisp/main.py monitor-identifier /path/to/map/file /path/to/command/file /path/to/env/file
+python mapdisp/main.py monitor-identifier
+       /path/to/map/file /path/to/command/file /path/to/env/file
```

Comment:
I'm not sure here how to deal with the long command line. Use noqa here.

Reply:
Files starting with docstrings having @package are command lines? There were some files that contained E501, and those have been fixed. Let me know if those should be restored to their originals. I will add # noqa: E501 to the docstring in that case.

Reply:
Ok, understood. I have modified mapdis/main.py according to your instructions. </br>

The majority of these types of fixes were @author lines. Work descriptions (next to email) sometimes exceed the length limit. In the event the team decides on docstring styles, I'm happy to make necessary changes. For now, I kept them the way I fixed it (keeping line limit without # noqa: E501).

Reply:
Yes, the author lines are a different story and yes, clearly, we need to decide on a comment style guide. Overall, PR is moving the comments in the right direction.